### ISSUE TITLE
missing link in documentation; before it's forgotten

### ISSUE THREAD

**Issue by CarpeDiemKopi** [TiddlyWiki/TiddlyWiki5#4186]
Click this link: [WidgetModules](https://tiddlywiki.com/dev/static/WidgetModules.html) 

and you get: 

![grafik](https://user-images.githubusercontent.com/32836136/63345189-776ede80-c352-11e9-949e-38258bf36c4a.png)

Click the link under cursor above:  

`https://tiddlywiki.com/dev/static/%2524%253A%252Fcore%252Fmodules%252Fwidgets%252Fwidget.js.html`

=> 404 File not found

**Comment by sycom** [comment]
I think you can find this issue on many pages of the static TiddlyWiki version since _system_ and _shadow_ tiddlers are not exported via the static generator.

eg : 
* https://tiddlywiki.com/dev/static/RootWidget%2520and%2520Rendering%2520Startup
* https://tiddlywiki.com/dev/static/Extended%2520Persistence
* ...

Should static also export all or some of the _shadow_ and _system_ tiddlers ? If some, which ones ? Or should it avoid render _shadow_ and _system_ links ?

**Comment by sukima** [comment]
> Should static also export all or some of the shadow and system tiddlers ? If some, which ones ? Or should it avoid render shadow and system links ?

No. because I use that on my personal blog for SEO reasons and I don't want all that (nor do my tiddlers link to shadow tiddlers)

I propose that the tiddlywiki5.com edition use an alternative template that could include those shadow tiddlers while the default for us user land folk keep it the way it is.

**Comment by sycom** [comment]
I guess there are issues at exporting every system tiddler and also for shadows, since they are part of others. This would create A LOT of barely readable pages.

Maybe you could try? Just modify [$:/core/save/all](https://github.com/Jermolene/TiddlyWiki5/blob/master/core/templates/save-all.tid) and change `[is[tiddler]]` to see what happens.
